doc/overlays.xml

```diff
@@ -30,10 +30,16 @@
       itself is given, then that is used.
      </para>
      <para>
-      This can be passed explicitly when importing nipxkgs, for example
+      This can be passed explicitly when importing nixpkgs, for example
       <literal>import &lt;nixpkgs> { overlays = [ overlay1 overlay2 ];
       }</literal>.
      </para>
+     <para>
+      Further overlays can be added by calling the
```

Comment:
nit: I think this deserve its own listitem, instead of being a note on one way to install overlays.

Reply:
A listitem didn't work, because the list was about the way nixpkgs looks up its overlays, so I have shuffled some things around to make it fit.